pkg/repo/migrations/helpers.go

```diff
@@ -11,8 +11,8 @@ import (
 	"github.com/libp2p/go-libp2p/core/peer"
```

Comment:
if/when we implement a v5 migration this package will need to be re-worked since it takes a dependency on the old config structure.